app/modules/settings/components/displayed_links.svelte

```diff
@@ -34,45 +42,86 @@
   const lazyUpdate = debounce(updateCustomProperties, 1000)
   function onSelectionChange () {
     customProperties = getPropertiesFromWebsitesNames(selectedWebsites)
+    selectedBibliographicDatabasesCount = intersection(bibliographicDatabasesNames, selectedWebsites).length
+    selectedSocialNetworksCount = intersection(socialNetworksNames, selectedWebsites).length
     lazyUpdate()
   }
   $: onChange(selectedWebsites, onSelectionChange)
+
+  function toggleAllSection ({ names, selectedCount }) {
+    if (selectedCount !== names.length) {
+      selectedWebsites = uniq(selectedWebsites.concat(names))
+    } else {
+      selectedWebsites = difference(selectedWebsites, names)
+    }
+  }
 </script>
 
-<fieldset>
-  <legend>{I18n('bibliographic databases')}</legend>
-  {#each Object.keys(bibliographicDatabases) as websiteName}
-    <label>
-      <input type="checkbox" bind:group={selectedWebsites} value={websiteName} />
-      {websiteName}
+{#if category == null || category === 'bibliographicDatabases'}
+  <section>
+    <label class="category">
+      <input
+        type="checkbox"
+        checked={selectedBibliographicDatabasesCount === bibliographicDatabasesNames.length}
+        indeterminate={selectedBibliographicDatabasesCount !== 0 && selectedBibliographicDatabasesCount !== bibliographicDatabasesNames.length}
+        on:click={() => toggleAllSection({ names: bibliographicDatabasesNames, selectedCount: selectedBibliographicDatabasesCount })}
```

Comment:
I recall a conversation where you where against this checkbox-that-modify-everything behavior. The reason behind was that it is not very accessible, as what is going to happen after clicking on the checkbox is left unknown to the user. Usecase: one crafts their ids customization, then discovers the general checkbox, and on their first click on it, they deletes their own customisation for good.

`Select All` button pattern seems to be the accepted solution since inventory browser implementation. And is now in other parts of the app (ie. items map, entity homonym selections). 
![2024-01-14_12-24](https://github.com/inventaire/inventaire-client/assets/5363918/73cd4afa-d33a-4026-9235-a3ec3b09c03e)

So, i think it would make sense, on a less central display than inventory browser (where space is less compact/important), to reuse this pattern, wouldnt it?

Reply:
> it respects a known convention

I couln't find this convention anywhere in the codebase though. On the contrary the "select all" pattern appears in more than the inventory browser (ie. map filters)

I would still be more comfortable with an already in use pattern (and more accessible (cf. my last comment)) to not have different display with same behavior.

If you want to continue with this unnamed checkbox, can the `input` element be extracted into its own component (to make reusable for map filters) ?

Reply:
> I couln't find this convention anywhere in the codebase though

I meant a known convention in general, but that has indeed not been used so far in the codebase

Reply:
This [implementation with the button and a dynamic label](https://www.sncf-voyageurs.com/fr/voyagez-avec-nous/horaires-et-itineraires/itineraires/) seems to work nicely

Reply:
I dont know how to continue this, i think its a mistake to present two different ways to display the same behavior. And sure, its minimalistic, but also more difficult to understand whats going on. This app is already complicated enough to make the user guess what will happen if checking the box.

You seem definitive, so i will swallow it, can we at least extract it in its own component to reuse it elsewhere ?

Reply:
I tried to see how to extract the input, but that was not conclusive: that looks like a case of painful-to-DRY-so-lets-copy.